kernel/poll.c

```diff
@@ -67,7 +67,6 @@ static inline int is_condition_met(struct k_poll_event *event, u32_t *state)
 		return 0;
 	default:
 		__ASSERT(0, "invalid event type (0x%x)\n", event->type);
-		break;
```

Comment:
So obviously that was a noop.  I don't disagree with removing it, but can you put unrelated cleanup in its own patch?  Having unrelated junk sitting around makes tasks like test reversions and bisections more likely to fail spuriously.  The commit message says this is a change to "tests", it shouldn't be hammering kernel files.

Reply:
Thanks for the input. I won't be able to change the code for next two days due to power shutdown issue here. I will remove the kernel changes from this commit and create a separate commit only with kernel code changes as soon i get access to my Linux machine.

Reply:
Updated as per suggestion. Created two different commits with kernel and test code separately. Kindly check. Thanks again.

Reply:
@andyross:
Kindly review the changes.